### PR TITLE
fix: issues reported from QA team.

### PR DESCRIPTION
**Pull request by shubhw12**
- While editing any font Font Name & Fallback should be on top  (Custom Font > edit font ) -    [image](https://drive.google.com/file/d/15170L-u2sBP5gQoHP3hc9VRmgJzEpGLw/view?usp=sharing) - `needs patch`
- Add Font Variation button should be at bottom- [image](https://drive.google.com/file/d/1mrGNBnSsTewfmjoYXeyI9uDlmpwi-_vf/view?usp=sharing) because if i have to add 5-6 variation I have to scroll up many times - `needs patch`
- Once variation is added  all `Remove` button should be active, it should be disabled only when one variation is present - [rec](https://vimeo.com/518567176/47cf64f44b) - `needs patch` - If this is time-consuming we can skip this from this release.
- In customizer getting a value as undefined - [rec](https://vimeo.com/518569475/a5ddf64bb7) - when font is set to Normal - `needs patch`

### CODE REVIEW

File: assets/js/bsf-custom-fonts.js [hunk 1]
```diff
@@ -64,11 +64,20 @@
 		},
 	}
 
-	var addButton = $('#repeater').find('.repeater-add-btn');
+	function remove_button_disble(){
```
Comment (premanshup) [inline assets/js/bsf-custom-fonts.js:67]: @shubhw12 Please update the string to `remove_button_disable`

File: includes/class-bsf-custom-fonts-admin.php [hunk 2]
```diff
@@ -214,17 +215,15 @@ public function add_new_taxonomy_repeater_data() {
...
-					'900'    => '900',
+					'100' => __( 'Thin 100', 'astra' ),
+					'200' => __( 'Extra-Light 200', 'astra' ),
+					'300' => __( 'Light 300', 'astra' ),
+					'400' => __( 'Normal 400', 'astra' ),
+					'500' => __( 'Medium 500', 'astra' ),
+					'600' => __( 'Semi-Bold 600', 'astra' ),
+					'700' => __( 'Bold 700', 'astra' ),
+					'800' => __( 'Extra-Bold 800', 'astra' ),
+					'900' => __( 'Ultra-Bold 900', 'astra' ),
```
Comment (premanshup) [inline includes/class-bsf-custom-fonts-admin.php:226]: @shubhw12 Replace `'astra'` with `'custom-fonts'`

File: includes/class-bsf-custom-fonts-admin.php [hunk 3]
```diff
@@ -307,17 +314,16 @@ public function edit_taxonomy_repeater_data( $key, $value ) {
...
+									'100' => __( 'Thin 100', 'astra' ),
+									'200' => __( 'Extra-Light 200', 'astra' ),
+									'300' => __( 'Light 300', 'astra' ),
+									'400' => __( 'Normal 400', 'astra' ),
+									'500' => __( 'Medium 500', 'astra' ),
+									'600' => __( 'Semi-Bold 600', 'astra' ),
+									'700' => __( 'Bold 700', 'astra' ),
+									'800' => __( 'Extra-Bold 800', 'astra' ),
+									'900' => __( 'Ultra-Bold 900', 'astra' ),
+
```
Comment (premanshup) [inline includes/class-bsf-custom-fonts-admin.php:326]: @shubhw12 Replace `'astra'` with `'custom-fonts'`

**Review by premanshup — CHANGES_REQUESTED** [review]

File: assets/js/bsf-custom-fonts.js (hunk 1 above)
Comment (shubhw12) [reply assets/js/bsf-custom-fonts.js]: @premanshup I have resolved this change

File: includes/class-bsf-custom-fonts-admin.php (hunk 2 above)
Comment (shubhw12) [reply includes/class-bsf-custom-fonts-admin.php]: @premanshup I have made these changes.

**Review by premanshup — APPROVED** [review]

**Pull request merged**